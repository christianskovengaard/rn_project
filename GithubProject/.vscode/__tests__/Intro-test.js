// __tests__/Intro-test.js
import React from 'react';
import List from '../../src/views/list'

import renderer from 'react-test-renderer';
import "isomorphic-fetch"
test('renders correctly', () => {
  const tree = renderer.create(<List />).toJSON();
  expect(tree).toMatchSnapshot();
});