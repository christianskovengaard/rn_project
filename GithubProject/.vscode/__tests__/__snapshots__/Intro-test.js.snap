// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={undefined}
  >
    Getting som Popular Github repos
  </Text>
</View>
`;
